build.gradle.kts

```diff
@@ -1,15 +1,15 @@
 
 import com.jetbrains.plugin.blockmap.core.BlockMap
 import groovy.lang.GroovyObject
+import org.jetbrains.intellij.tasks.RunPluginVerifierTask
+import org.jfrog.gradle.plugin.artifactory.dsl.PublisherConfig
```

Comment:
I think it's supposed to be ordered alphabetically, do you use the right codestyle?